internal/cli/main.go

```diff
@@ -405,6 +405,33 @@ func Commands(
 			}, nil
 		},
 
+		"trigger": func() (cli.Command, error) {
+			return &helpCommand{
+				SynopsisText: helpText["trigger"][0],
+				HelpText:     helpText["trigger"][1],
+			}, nil
+		},
+		"trigger apply": func() (cli.Command, error) {
```

Comment:
"apply" feels a bit weird for triggers, I wonder if we should split into "create" and "update". Triggers don't really feel like some desired state operation... even if internally we support an upsert.

Reply:
I started down the path with `create` and `update` (see git commit history for that) but wasn't really sure how to handle it cleanly without a bunch of duplicated code or a shared lib for handling operations and flags. If you have ideas I'm all ears!

Reply:
The way we've done it before is make it the same command struct, and just put a `Update bool` on it and then switch on that logic in it. So most of the logic is shared, but ie. if `Update == true` then we require an ID. And we can also change some of the help output that way.

Reply:
Got it! Good idea. Got that updated now.